src/structure/cacheManager.ts

```diff
@@ -0,0 +1,44 @@
+import { Collection } from 'discord.js';
+
+const EXPIRED_MS = 8.64e7; // 24 hours
```

Comment:
Let's make this 10 mins to allow for when descriptions/labels are updated

Reply:
10 mins = `6e5` ms